test/libsolidity/semanticTests/constructor_inheritance_init_order.sol

```diff
@@ -11,9 +11,9 @@ contract B is A {
     uint public y = f();
 }
 // ====
-// compileToEwasm: also
 // compileViaYul: true
 // ----
 // constructor() ->
 // gas irOptimized: 120936
+// gas legacy: 115102
```

Comment:
That's a bit weird, not sure where that's coming from :-).

Reply:
hmm.. true.. I added this manually yesterday, because it looked like that it is tested somehow - I think I will just revert the change